### PR TITLE
chore: add tweets about open graph

### PR DESCRIPTION
**Pull request by daKmoR**
## What I did

1.

### CODE REVIEW

File: tweets/2022-03-31/2022-03-31-tweets.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+2. Open the browser on `http://localhost:8000/index.opengraph.html`
+3. Open Dev Tools and set width to `1200px` and height to `628px` and zoom to `50%`
+
+We can now adjust the HTML until we are satisfied with the result.
+
+![Basic Open Graph image](./01-basic.png)
+
+---
+
+One other important thing to note is that it is highly recommended to use a web font.
```
Comment (Matsuuu) [inline tweets/2022-03-31/2022-03-31-tweets.md:110]: I don't know if this tweet is 100% necessary in here. It might be good to keep as part of the guides, but in a tweet thread, it just makes it feel more like a tutorial than a showcase

File: tweets/2022-03-31/2022-03-31-tweets.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+Once you have a multiple pages maybe even with multiple different open graph layouts it is hard to check for if all images are "correct" in the sense of the text is not too long/short and images are not overlapping.
+
+But fear not you can also create your own overview of all pages.
+
+---
+
+The Open Graph Overview features
+
+➡️ Uses Rockets PageTree to get a list of all pages
```
Comment (Matsuuu) [inline tweets/2022-03-31/2022-03-31-tweets.md:240]: I would maybe rephrase these into more "doing" terms.

```
The Open Graph Overview features

- Using Rockets PageTree to get a list of all pages
- Showing Open Graph pages
- Opening URL's
- Opening Open Graph pages
- Opening source files in VSCode
```

File: tweets/2022-03-31/2022-03-31-tweets.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+
+Now let's setup an Open Graph image for our site.
+
+1. Add `export const openGraphLayout = data => ...` to `recursive.data.js` so it gets injected into every page
+2. On the page itself we export some data that gets passed to the layout
+3. Define the `absoluteBaseUrl` in our rocket config as it needs to be an absolute url
+
+👉 `index.rocket.js`
+
+```js
```
Comment (Matsuuu) [inline tweets/2022-03-31/2022-03-31-tweets.md:58]: Maybe swap the order of this code block and the next one, as they are referenced in the opposite order above

File: tweets/2022-03-31/2022-03-31-tweets.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+
+Once we have an image we add it as a meta tag to our HTML.
+
+---
+
+You may thinks "what is this fuss about?" create this image once and you are done.
+
+Not so fast! Every page should have a unique image and it should reflect the content of the page.
+
+We wanted to stand out on social media so we will need to go the extra mile.
```
Comment (Matsuuu) [inline tweets/2022-03-31/2022-03-31-tweets.md:36]: Could we add an example here? e.g. "Rocket Docs - Creating an open graph image" -OG image as an embed, to show the "unique image"

**Review by Matsuuu — APPROVED** [review]
A few things to touch up on, but otherwise I think it's a good thread! :)

File: tweets/2022-03-31/2022-03-31-tweets.md (hunk 1 above)
Comment (daKmoR) [reply tweets/2022-03-31/2022-03-31-tweets.md:110]: right - will tackle that separately at some point 👍
